meta-refkit/conf/distro/refkit.conf

```diff
@@ -77,6 +77,15 @@ PACKAGECONFIG_remove_pn-pulseaudio = "avahi"
 # leave out readline, gdbm, and db
 PACKAGECONFIG_pn-python3 = ""
 
+# drop python2 support
+PACKAGECONFIG_remove_pn-opkg-utils = "python"
```

Comment:
There's no such package in the image manifests. But the supported recipes check fails if it's missing from refkit-supported-recipes.txt
@pohly could you please help with this? Perhaps it gets pulled because of update-alternatives in coreutils.